src/containers/App/App.tsx

```diff
@@ -46,28 +40,9 @@ const App = ({ actions }: AppProps) => {
     return function cleanup() {
       window.removeEventListener('resize', actions.updateViewportDimensions)
       window.removeEventListener('scroll', actions.onScroll)
-
-      socket.close()
-      if (socket.p2pSocket !== undefined) {
-        socket.p2pSocket.close()
-      }
```

Comment:
Why was this part removed?

Reply:
It was moved inside the new `<SocketProvider>` component which handles all setup and teardown of the SocketContext.